values.yaml

```diff
@@ -14,6 +14,7 @@ dcOdbcImagePullPolicy: IfNotPresent
 namespaced:
 revisionHistoryLimit: 10
 resultsQueue: JOB_RESULTS
+javaOpts: -Dloader.path=/opt/Actian/engine/runtime/di9/jars/opt/client-sdk-local/di-artifacts-12.0.0-31.jar,/opt/Actian/engine/runtime/di9/jars/opt/client-sdk-local/di-client-sdk-12.0.0-31.jar,/opt/Actian/engine/runtime/di9/jars/opt/client-sdk-local/di-design-sdk-12.0.0-31.jar,/opt/Actian/engine/runtime/di9/jars/opt/client-sdk-local/jcore-12.0.0-31.jar,/opt/Actian/engine/runtime/di9/jars/opt/client-sdk-local/package-explorers-12.0.0-31.jar
```

Comment:
Added 4 specific jars with default engine version, however we are going to override this param environement specific values.yaml file.